changelog.md

```diff
@@ -1,4 +1,5 @@
 # Future
+- [CHANGED] MSSQL `removeColumn` query to pass options to `sequelize.query` [#7139](https://github.com/sequelize/sequelize/pull/7193)
```

Comment:
[FIXED] Properly pass options to `sequelize.query` in `removeColumn` [MSSQL] [#7139](https://github.com/sequelize/sequelize/pull/7193)